test/integration/ferc1_eia_train_test.py

```diff
@@ -27,37 +27,13 @@
     generate_all_override_spreadsheets,
     validate_override_fixes,
 )
+from pudl.helpers import get_parquet_table
 
 logger = logging.getLogger(__name__)
 
-if os.getenv("GITHUB_ACTIONS", False):
-    pytest.skip(
-        reason="As of 2025-05-25 this test is causing the GitHub runner to run out of memory.",
-        allow_module_level=True,
-    )
-
-
-@pytest.fixture(scope="module")
-def utils_eia860(fast_out_annual):
-    """The utils_eia860 output table."""
-    return fast_out_annual.utils_eia860()
-
-
-@pytest.fixture(scope="module")
-def plant_parts_eia(fast_out_annual):
-    """The plant_parts_eia output table."""
-    return fast_out_annual.plant_parts_eia().reset_index()
-
 
 @pytest.fixture(scope="module")
-def eia_ferc1(fast_out_annual):
-    """The eia_ferc1 output table."""
-    # the pudl_out name here is the old, non-alphabetized ordering.
-    return fast_out_annual.ferc1_eia()
```

Comment:
Now I read these dataframes from parquet directly, inside the test functions. This results in some duplicate reads, since the same data is used in more than one test, but they're fast.